tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -170,6 +174,7 @@ spec:
         - "--strict=false"
         - "--show-successes"
         - "--effective-time=$(params.EFFECTIVE_TIME)"
+        - "--extra-rule-data=$(params.EXTRA_RULE_DATA)"
```

Comment:
```suggestion
        - "--extra-rule-data='$(params.EXTRA_RULE_DATA)'"
```

Reply:
I think this will help support a key or value with a space, e.g. `--extra-rule-data='foo=hi there,bar=123'` might work, but `--extra-rule-data=foo=hi there,bar=123` will throw errors.

Reply:
Okay, I guess I don't know how to quote these things..

Acceptance test has this:

```
- Snapshot - 2
+ Received + 2

@@ -85,7 +85,7 @@

      - github.com/enterprise-contract/ec-policies//policy/release
      - github.com/enterprise-contract/ec-policies//policy/lib
      ruleData:
-       key1: value1
-       key2: value2
+       '''key1': value1
+       key2: value2'
```

Reply:
@simonbaird Is that acceptable behavior? Feels like it could cause issues if that quoting makes the key/value different than intended. Or it could just be an anomaly in the report only?